hack/test-bazel-build-tarball.sh

```diff
@@ -3,5 +3,11 @@ if [ "$IS_CONTAINER" != "" ]; then
   set -x
   bazel --output_base=/tmp build "$@" tarball
 else
-  docker run -e IS_CONTAINER='TRUE' --rm -v "$PWD":"$PWD" -v /tmp:/tmp:rw -w "$PWD" quay.io/coreos/tectonic-builder:bazel-v0.3 ./hack/test-bazel-build-tarball.sh
+  docker run --rm \
+    --env IS_CONTAINER=TRUE \
+    --volume "${PWD}:${PWD}:z" \
+    --volume /tmp:/tmp:z \
```

Comment:
This seems dangerous...  You might be relabling something on your local host that another process would be using...

Reply:
> This seems dangerous...

Are you only concerned about the host `/tmp`?  Or are you also concerned about `${PWD}`?  I expect we can get this working without the `/tmp` mount, but I don't see a way around the `${PWD}` mount.

Reply:
only /tmp.  GDM, X, systemd, and other things put files in /tmp and expect to be able to access them.  If we use `:z` when mounting /tmp we will be relabeling those files and might remove permission for the proper owner to work.  We will also be relabeling /tmp itself and messing up the labels that other programs create in /tmp...

${PWD} won't likely be used by (many) other programs in this case. So relabeling seems quite reasonable.

Reply:
Does the container need some content in /tmp?

Reply:
> Does the container need some content in /tmp?

We may be able to drop the mount, since [we didn't have it before][1].  @sallyom, do you remember why you added it?

[1]: https://github.com/openshift/installer/pull/117/files#diff-354f30a63fb0907d4ad57269548329e3L30